config/config.go

```diff
@@ -19,6 +19,7 @@ type Config struct {
 	fallbackURL string
 
 	General struct {
+		ApplicationName              string `public:"true" info:"Sets the name used throughout the application."`
```

Comment:
We should include a bit about how this is used
```suggestion
		ApplicationName              string `public:"true" info:"Override the name used in messaging and page titles. If unset 'GoAlert' will be used."`
```